docs/pages/datapoint.js

```diff
@@ -39,12 +47,13 @@ export default function DatapointPage({ generatedDocGen }: {| generatedDocGen: D
           />
         </MainSection.Subsection>
       </MainSection>
+
       <MainSection name="Best practices">
         <MainSection.Subsection columns={2}>
           <MainSection.Card
             cardSize="md"
             type="do"
-            description="Whenever possible, use the full number with separators for groups of thousands to ensure clarity across all languages."
+            description="Whenever possible, use the full number with locale-specific separators to ensure clarity across all languages."
```

Comment:
separators are another thing that changes by locale:
https://phrase.com/blog/posts/number-localization/#Separators_Breaking_Up_Large_Numbers